tools/boostsecurity

```diff
@@ -0,0 +1,69 @@
+{
+      "name": "Boostsecurity.io",
+      "publisher": "Boostsecurity.io",
+      "description": "The zero friction platform for AppSec Testing and Application Security Posture Management (ASPM) securing both Software Supply Chains and AI Development.",
```

Comment:
this sounds like marketing language, not like a helpful descriotion what this services is for.
please improve.